tools/nixpkgs-hammer

```diff
@@ -160,12 +241,21 @@ def main(args):
                       severity = "error";
                       link = false;
                     }} ];
-                    position = null;
+                    location = null;
+                    outputPath = null;
+                    drvPath = null;
                   }}
                 else
                   {{
                     report = if maybeReport.success then maybeReport.value else [];
-                    position = result.value.meta.position;
+                    location = let
+                      posSplit = builtins.split ":" (result.value.meta.position or ":");
+                    in {{
+                      file = builtins.elemAt posSplit 0;
+                      line = builtins.elemAt posSplit 2;
```

Comment:
Should not this be 1? Also what happens when there is no `meta.position`? Will these be empty strings?

Reply:
Ye. Or I can convert them to `null`s.

Reply:
Converting the whole position to `null` like in the other parts sounds good.

Reply:
https://github.com/jtojnar/nixpkgs-hammering/issues/72 is a good test case for this, it seems.

Reply:
Done (and it fixes #72)